src/applications/personalization/profile360/components/PhoneSection.jsx

```diff
@@ -60,8 +60,8 @@ class EditPhoneModal extends React.Component {
 
             <ErrorableTextInput
               label="Number"
-              field={{ value: field.value.phoneNumber, dirty: false }}
-              onValueChange={this.onChange('phoneNumber')}
+              field={{ value: field.value.inputPhoneNumber, dirty: false }}
```

Comment:
Added this to handle the full phone number that users enter. When submitting to vet360 this input is split into Area Code and Phone Number components